packaging/Dockerfile.arm64

```diff
@@ -1,10 +1,10 @@
-FROM --platform=linux/amd64 alpine:3.13 AS build
+FROM --platform=linux/arm64 alpine:3.13
 LABEL maintainer="Team Teapot @ Zalando SE <team-teapot@zalando.de>"
 RUN apk --no-cache add ca-certificates && update-ca-certificates
-
-FROM --platform=linux/arm64 alpine:3.13
-COPY --from=build /etc/ssl/certs/ /etc/ssl/certs/
```

Comment:
I guess this was there to not `RUN apk --no-cache add ca-certificates && update-ca-certificates` on the result image.

Reply:
It was. I added it recently to make a previous try work: https://github.com/zalando/skipper/pull/1954/files#diff-d644f844087e1cb2b3e229fef8aec05cfa86372250e0dfae56a9c87140c356f1.

This just restores the original version.